plugins/zfs/zfs_list

```diff
@@ -75,9 +77,9 @@ fi
 
 for i in `zfs list -Hp | awk '{print $1}'`; do
 	values=( $(zfs get -p usedbydataset,usedbychildren,usedbysnapshots,usedbyrefreservation,available,quota $i | awk 'BEGIN {total=0;} { if( NR==1 ) next; } !/quota/ {total=total+$3;} {print $3} END{print total;}') )
-	fsname=`echo $i | sed 's/\//__/g'`
+	fsname=$(clean_fieldname $(echo "$i" | sed 's/\//__/g'))
 
-echo <<EOF "multigraph zfs_list_$fsname
+	echo <<EOF "multigraph zfs_list_$fsname
```

Comment:
Did you change the indentation here on purpose?

Reply:
Yes to align with the same echo as done with the echo in the for loop as done above.
It'll also be easier to read, as it'll be a bit less confusing I suppose for someone who is not a bash expert.

Since this is a bugfix, and if you don't want to see that indentation fix in this bugfix, I could revert and just leave it as it won't change anything besides making it structure wise better.

Reply:
OK. So it just looked weird on first sight for me. Thank you for your explanation.